packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -225,6 +230,12 @@ export const getCFDAccount = ({
     return CFD_text[cfd_account_key as keyof typeof CFD_text];
 };
 
+export const getMT5Icon = ({ market_type, is_eu }: TGetMT5Icon) => {
+    if (market_type === 'all' && !is_eu) return 'SwapFree';
+    if (market_type === 'financial' && is_eu) return 'CFDs';
```

Comment:
Can you remove the hardcoded values?
`all`, `financial`, `SwapFree` and `CFDs`.

Reply:
It is included in the other PR for Hard-coded one will need to wait for merge after that we can remove these values otherwise we still have to depend on shared package